OCPBUGS#37577: Add a note for identity-provider-arn (#109763)
openshift/openshift-docs: security/cert_manager_operator/cert-manager-operator-install.adoc

[role="_abstract"]
The {cert-manager-operator} is not installed in {product-title} by default. You can install the {cert-manager-operator} by using the web console.

[NOTE]
====

> The {cert-manager-operator} sets the `features.operators.openshift.io/token-auth-aws`, `features.operators.openshift.io/token-auth-azure`, and `features.operators.openshift.io/token-auth-gcp` annotations in the `ClusterServiceVersion custom resource` of the Operator. The {product-title} web console requires the credential details when these annotations are set. Currently, the Operator does not use the values collected by the OpenShift web console and you can provide any value when asked for the input. For example, when installing on the managed {product-title} cluster, the `identity-provider-arn` is asked and any value can be provided to proceed.

Reviewer: ```suggestion
The {cert-manager-operator} sets the `features.operators.openshift.io/token-auth-aws`, `features.operators.openshift.io/token-auth-azure`, and `features.operators.openshift.io/token-auth-gcp` annotations in the `ClusterServiceVersion` custom resource of the Operator. The {product-title} web console requires the credential details when these annotations are set. Currently, the Operator does not use the values collected by the OpenShift web console and you can provide any value when asked for the input. For example, when installing on the managed {product-title} cluster, the `identity-provider-arn` is asked and any value can be provided to proceed.
```

Reviewer: @aksjoshi89 one nit. DM me when you've made this minor change and I'll merge it for you.

> The operator sets `features.operators.openshift.io/token-auth-aws`, `features.operators.openshift.io/token-auth-azure`, `features.operators.openshift.io/token-auth-gcp` annotations in the operator CSV, and console asks to provide the relevant credentials details when these annotations are set. Currently, the operator does not make use of the values collected by the console and users should follow https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift. Users can provide any dummy value when asked for the input. For example, when installing on ROSA, the `identity-provider-arn` is asked and any value can be provided to proceed.

Reviewer 2: Yeah, these annotations are set on the operator's CSV(cluster service version) in the bundle, which gets carried over to the operator deployment object.

It should be `OpenShift Console`.

> The {cert-manager-operator} sets `features.operators.openshift.io/token-auth-aws`, `features.operators.openshift.io/token-auth-azure`, and `features.operators.openshift.io/token-auth-gcp` annotations in the CSV of the Operator. The {product-title} console asks to provide the relevant credentials details when these annotations are set. Currently, the Operator does not make use of the values collected by the console and users should follow https://docs.redhat.com/en/documentation/openshift_container_platform/4.20/html/security_and_compliance/cert-manager-operator-for-red-hat-openshift. Users can provide any dummy value when asked for the input. For example, when installing on ROSA, the `identity-provider-arn` is asked and any value can be provided to proceed.

Reviewer 3: ```suggestion
The {cert-manager-operator} sets the `features.operators.openshift.io/token-auth-aws`, `features.operators.openshift.io/token-auth-azure`, and `features.operators.openshift.io/token-auth-gcp` annotations in the `ClusterServiceVersion custom resource` of the Operator. The {product-title} web console requires the credential details when these annotations are set. Currently, the Operator does not use the values collected by the OpenShift web console and you can provide any value when asked for the input. 
```
-use definite articles (the)
-the CSV is an object (be sure to look this kind of thing up)
-it is the OpenShift web console (there are other consoles, confirm with SME which this is supposed to be)
-the link cannot be here and was improperly formatted; see examples in the repo
-the writing here was not minimal and too verbose
-ROSA is not the proper name for AWS managed OpenShift; also, if this doc is for regular OpenShift, don't give a ROSA example here
